### PR TITLE
4.2 antora files separate manuals

### PR DESCRIPTION
**Pull request by recrwplay**
Adds the required antora files to generate separate drivers manuals

### CODE REVIEW

File: dotnetManual/antora/antora.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: dotnet-manual
+title: Neo4j Dotnet Manual
```
Comment (davidoliverSP2) [inline dotnetManual/antora/antora.yml]: do these titles surface in the manual?

if so, I think it should be **The Neo4j .NET Driver Manual**

**Review by davidoliverSP2 — APPROVED** [review]
👍  

thanks @recrwplay - just a quick question about whether the titles in the antora.yml files surface in the manuals anywhere (I was thinking perhaps in the browser tab?)

File: dotnetManual/antora/antora.yml (hunk 1 above)
Comment (recrwplay) [reply dotnetManual/antora/antora.yml]: Good point. This isn't an attribute that would be used to generate title text from our asciidoc, but it is potentially used by Antora in a few places (for example, if we used the version picker to display the list of available manuals as well as versions of those manuals, this title attribute would be displayed in that list.

Comment (recrwplay) [reply dotnetManual/antora/antora.yml]: Also in the browser tab - good spot!

Comment (recrwplay) [reply dotnetManual/antora/antora.yml]: All title attributes updated to match the manual's index page title

File: jsManual/antora/antora.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: javascript-manual
+title: Neo4j JavaScript Driver Manual
+version: '4.2'
```
Comment (davidoliverSP2) [inline jsManual/antora/antora.yml]: The version here is 4.2, while the others are 4.2-preview

Comment (recrwplay) [reply jsManual/antora/antora.yml]: Gah. It was also missing `prerelease: true`

**Pull request merged**